lib/cisco_node_utils/cmd_ref/fabricpath_topology.yaml

```diff
@@ -0,0 +1,35 @@
+# fabricpath_topology
+---
+# Fabricpath feature is not available on N3K and N9K
+_exclude: [/N3/, /N9K/]
+
+all_topos:
+  multiple:
+  config_get: "show fabricpath topology"
```

Comment:
The show runn cannot retrieve the 'default topology' membership details (which is what will be existing in most customer configs), nor the default description.

From: Chris Van Heuveln <notifications@github.com<mailto:notifications@github.com>>
Reply-To: cisco/cisco-network-node-utils <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Monday, November 23, 2015 at 1:34 PM
To: cisco/cisco-network-node-utils <cisco-network-node-utils@noreply.github.com<mailto:cisco-network-node-utils@noreply.github.com>>
Cc: Cisco Employee <dcherian@cisco.com<mailto:dcherian@cisco.com>>
Subject: Re: [cisco-network-node-utils] Collapse feature/fabricpath into develop (#102)


In lib/cisco_node_utils/cmd_ref/fabricpath_topology.yaml<https://github.com/cisco/cisco-network-node-utils/pull/102#discussion_r45664046>:

> @@ -0,0 +1,35 @@
> +# fabricpath_topology
> +---
> +# Fabricpath feature is not available on N3K and N9K
> +_exclude: [/N3/, /N9K/]
> +
> +all_topos:
> +  multiple:
> +  config_get: "show fabricpath topology"


Use show runn, not show fabricpath topo...

# sh run fabricpath topology all
fabricpath topology 2
  description foo
  member vlan 3


-
Reply to this email directly or view it on GitHub<https://github.com/cisco/cisco-network-node-utils/pull/102/files#r45664046>.